docs/reading-list.md

```diff
@@ -0,0 +1,164 @@
+https://youtu.be/bEZcodengwk)
+
+[Video] [NoOps, by Kelsey Hightower (DepOpsDays 2016)](
+https://youtu.be/ajT90pC3ris)
+
+[Video] [The mind behind Linux (TED interview)](
+https://youtu.be/o8NPllzkFhE)
+
+[Tutorial] [What happens when...](
+https://github.com/alex/what-happens-when)
+
+[Article] [An Interview With Linus Torvalds](
+https://techcrunch.com/2012/04/19/
+an-interview-with-millenium-technology-prize-finalist-linus-torvalds/)
+
+[Article] [Effective Learning Strategies for Programmers](
+http://akaptur.com/blog/2015/10/10/
+effective-learning-strategies-for-programmers/)
+
+[Article] [Readme Driven Development](
+http://tom.preston-werner.com/2010/08/23/readme-driven-development.html)
+
+[Article] [Systematic Debugging](
+http://akaptur.com/blog/2013/07/24/systematic-debugging/)
+
+[Paper] [Floating-Point Arithmetic](
+https://docs.oracle.com/cd/E19957-01/806-3568/ncg_goldberg.html)
+
+## Python
+
+[Video] [Intro to Python for beginners, by Jessica McKellar (PyCon 2013)](
+https://youtu.be/rkx5_MRAV3A)
+
+[Video] [Breaking the rules, by Jessica McKellar (PyCon Sweden)](
+https://youtu.be/C0fnHhY9UOc)
+
+[Video] [Build & break a Python sandbox, by Jessica McKellar (PyCon 2014)](
+http://pyvideo.org/pycon-us-2014/
+building-and-breaking-a-python-sandbox.html)
+
+[Video] [Cache me if you can, by Guillaume Ardaud (PyCon 2014)](
+http://pyvideo.org/pycon-us-2014/
+cache-me-if-you-can-memcached-caching-patterns.html)
+
+[Video] [Loop like a native, by Ned Batchelder (PyCon 2013)](
+https://youtu.be/EnSu9hHGq5o)
+
+[Video] [Modern Dictionaries, by Raymond Hettinger (SF Python)](
+https://youtu.be/p33CVV29OG8)
+
+[Video] [Python Language, by Guido van Rossum (PyCon 2016)](
+https://youtu.be/YgtL4S7Hrwo)
+
+[Video] [The Mighty Dictionary, by Brandon Rhodes (PyCon 2010)](
+http://pyvideo.org/pycon-us-2010/the-mighty-dictionary-55.html)
+
+[Tutorial] [Code Like a Pythonista: Idiomatic Python](
+http://python.net/~goodger/projects/pycon/2007/idiomatic/handout.html)
+
+[Article] [Static types in Python, oh my(py)!](
+http://blog.zulip.org/2016/10/13/static-types-in-python-oh-mypy/)
+
+[Guide] [The Hitchhiker’s Guide to Python!](
+http://docs.python-guide.org/en/latest/)
+
+## C/C++
+
+[Book] [The C Programming Language](
+https://www.ime.usp.br/~pf/Kernighan-Ritchie/C-Programming-Ebook.pdf)
+
+## Java/Android
+
+[MOOC] [Android Development for Beginners](
+https://www.udacity.com/course/android-development-for-beginners--ud837)
+
+## JavaScript/ECMAScript
+
+[Tutorial] [clean-code-javascript Software engineering principles](
+https://github.com/ryanmcdermott/clean-code-javascript)
+
+[Slides] [TypeScript vs. CoffeeScript vs. ES6](
+http://www.slideshare.net/NeilGreen1/type-script-vs-coffeescript-vs-es6)
+
+## Git/Version Control Systems (VCS)
+
+[Article] [Git tips](
+http://blog.sidhant.me/post/155311539806/
+stuff-that-i-learnt-in-git-while-working-with)
+
+## Computer Science & algorithms
```

Comment:
Maybe Computer Science/Algorithms? To keep it consistent with the previous headers.